test/e2e/accounts/snap-account-transfers.spec.ts

```diff
@@ -59,6 +59,9 @@ describe('Snap Account Transfers', function (this: Suite) {
     }
 
     if (isAsyncFlow) {
+      await driver.waitForElementNotPresent({
+        text: 'Please complete the transaction on the Snap.',
+      });
```

Comment:
It looks like we have to wait for this message to be gone before moving over with the "async flow", otherwise after navigating to the main screen we'll not receive the alert with the `Go to site` button to continue the test.

This seems to be related to the fact that interactions with Snaps take longer when using `@metamask/browser-passworder` to encrypt / decrypt, which is expectable.